iOS/Scripts/Sources/Scripty/main.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import ScriptyCore
+
+do {
+  try Scripty.run()
+} catch {
+  debugPrint("Error: \(error)")
+  exit(1)
+}
```

Comment:
This is the actual executable which is run - it grabs everything else in `ScriptyCore` as a library